src/set_theory/ordinal/basic.lean

```diff
@@ -504,14 +504,21 @@ theorem type_eq {α β} {r : α → α → Prop} {s : β → β → Prop}
   [is_well_order α r] [is_well_order β s] :
   type r = type s ↔ nonempty (r ≃r s) := quotient.eq
 
-@[simp] lemma type_lt (o : ordinal) : type ((<) : o.out.α → o.out.α → Prop) = o :=
+@[simp] theorem type_lt (o : ordinal) : type ((<) : o.out.α → o.out.α → Prop) = o :=
 begin
   change type o.out.r = _,
   refine eq.trans _ (quotient.out_eq o),
   cases quotient.out o,
   refl
 end
 
+theorem injective_out : injective (@quotient.out _ ordinal.is_equivalent) :=
+λ a b h, begin
+  apply_fun (λ o, type o.r) at h,
+  change type (<) = type (<) at h,
+  rwa [type_lt, type_lt] at h
+end
```

Comment:
Same comment as #13562, this is true more generally for `quotient.out`

Reply:
I've repurposed the PR for just the small tweaks